src/components/riskmap/RiskMap.tsx

```diff
@@ -423,19 +447,18 @@ const RiskMap = ({title, filePath, zoom}: RiskMapProps) => {
     selectedStartTime,
     selectedEndTime,
   ]);
-  
+
   const finalListWithDirection = useMemo(() => {
-    if (!selectedDirections.includes('Both')) {
-      return filteredAccidentData.filter(
-        (location) => {
-            return selectedDirections.some((direction => location.Zijde === direction))
-        }
-      );
-    }
-    else {
-      return filteredAccidentData
+    if (!selectedDirections.includes("Both")) {
+      return filteredAccidentData.filter((location) => {
+        return selectedDirections.some((direction) =>
+          location.Zijde.startsWith(direction)
```

Comment:
I am curious why to replace `location.Zijde === direction` to `location.Zijde.startsWith(direction)`?

Reply:
It's because while some data has "L" and "R", other data (like the updated file) has "Li" and "Re".